_posts/2016-01-20-set-up-and-run-hubot-in-non-heroku-server.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Setting up and running Hubot in a non-Heroku server"
```

Comment:
@mauro-oto "Setting up Hubot without Heroku" or "Setting up Hubot outside of Heroku"